examples/tutorial/vertical_federated_learning/cifar10-splitnn/job_configs/cifar10_splitnn/server/config/config_fed_server.json

```diff
@@ -0,0 +1,47 @@
+{
+  "format_version": 2,
+
+  "num_rounds": 15625,
+  "batch_size": 64,
+
+  "server": {
+    "heart_beat_timeout": 600
+  },
+  "task_data_filters": [],
+  "task_result_filters": [],
+  "components": [
+    {
+      "id": "persistor",
+      "path": "nvflare.app_common.pt.pt_file_model_persistor.PTFileModelPersistor",
```

Comment:
We also need to change these places to use "app_opt" for pt files.
But this could be done in a follow up PR.

Reply:
Sure. I changed it.